testsuite/MDAnalysisTests/analysis/test_contacts.py

```diff
@@ -315,7 +317,16 @@ def test_distance_box(self, pbc, expected):
         r = contacts.Contacts(u, select=(sel_acidic, sel_basic),
                         refgroup=(acidic, basic), radius=6.0, pbc=pbc)
         r.run()
-        assert_array_almost_equal(r.timeseries[:, 1], expected)
+        assert_array_almost_equal(r.results.timeseries[:, 1], expected)
+
+    def test_warn_deprecated_attr(self, universe):
+        """Test for warning message emitted on using deprecated `timeseries`
+        attribute"""
+        CA1 = self._run_Contacts(universe, stop=1)
+        wmsg = "The `timeseries` attribute was deprecated in MDAnalysis"
+        with pytest.warns(DeprecationWarning, match=wmsg):
+            assert_equal(CA1.timeseries, CA1.results.timeseries)
```

Comment:
good test: check for message and equality

Reply:
We're obviously both being too efficient 🙃